### PR TITLE
fix react warning: pass boolean, not string

### LINKED ISSUE #2789: Fix React warnings in src/app/containers/PageHandlers/withError/index.test.jsx
**Describe the bug**
There are React warnings in unit tests.

**To Reproduce**

1. Go to simorgh in terminal
2. Run npm run test:unit
3. Observe React warnings:

```
    console.log src/testHelpers/setupTests.js:19
      Test failed because too many React warnings were detected. Please fix the following:
      Warning: Failed prop type: Invalid prop `error` of type `string` supplied to `ErrorContainer`, expected `boolean`.
          in ErrorContainer
    console.error src/testHelpers/setupTests.js:30
      Warning: Failed prop type: Invalid prop `error` of type `string` supplied to `ErrorContainer`, expected `boolean`.
          in ErrorContainer
```

**Expected behaviour**

No React warning for test suite.

**Testing notes**

No testing required.

### PR DESCRIPTION
**Pull request by thekp**
Resolves #2789

**Overall change:** _update `withError` test, to pass a boolean as prop._

**Code changes:**

- _pass boolean as prop, instead of a string

---

- [x] I have assigned myself to this PR and the corresponding issues
- [ ] Tests added for new features
- [ ] Test engineer approval

### CODE REVIEW

**Review by chris-hinds — APPROVED** [review]
👍

**Review by dr3 — CHANGES_REQUESTED** [review]
I actually think the HOC is wrong here, https://github.com/bbc/simorgh/blob/latest/src/app/containers/App/App.jsx#L68 doesn't set `error` to a boolean, its set to the exception thrown in the catch

**Review by oluoluoxenfree — APPROVED** [review]

File: src/app/containers/PageHandlers/withError/index.test.jsx [hunk 1]
```diff
@@ -9,14 +9,14 @@ describe('withError HOC', () => {
...
-      <ErrorHOC error="error handling at its best" />,
+      <ErrorHOC error={new Error('This is an error.')} />,
     );
   });
 
   describe('with no error', () => {
     shouldShallowMatchSnapshot(
       `should return the passed in component`,
-      <ErrorHOC error={null} />,
+      <ErrorHOC error />,
```
Comment (dr3) [inline src/app/containers/PageHandlers/withError/index.test.jsx]:
```suggestion
      <ErrorHOC />,
```
`<ErrorHOC error />` is the same as `<ErrorHOC error={true} />`

Comment (thekp) [reply src/app/containers/PageHandlers/withError/index.test.jsx]: good shout (snapshots were wrong as well)

**Review by dr3 — APPROVED** [review]

File: src/app/containers/PageHandlers/withError/index.jsx [hunk 2]
```diff
@@ -9,11 +9,11 @@ const WithError = Component => {
   };
 
   ErrorContainer.propTypes = {
-    error: bool,
+    error: shape({ message: string }),
```
Comment (dr3) [inline src/app/containers/PageHandlers/withError/index.jsx]:
```suggestion
    error: shape({ message: string.isRequired }),
```

**Review by dr3 — APPROVED** [review]

**Pull request merged**